src/test/kotlin/graphics/scenery/tests/examples/volumes/TransferFunctionEditorExample.kt

```diff
@@ -0,0 +1,144 @@
+        v.setTransferFunctionRange(0.0f, 65000.0f)
+        scene.addChild(v)
+
+
+        val bridge = SwingBridgeFrame("1DTransferFunctionEditor")
+        val tfUI = TransferFunctionUI(650, 550, v, bridge)
+        val swingUiNode = tfUI.mainFrame.uiNode
+        swingUiNode.spatial() {
+            position = Vector3f(2f,0f,0f)
+        }
+
+        scene.addChild(swingUiNode)
+    }
+
+    override fun inputSetup() {
+        super.inputSetup()
+
+        val debugRaycast = false
+        inputHandler?.addBehaviour(
+            "ctrlClickObject", object : ClickBehaviour {
+                override fun click(x: Int, y: Int) {
+                    val ray = cam.getNodesForScreenSpacePosition(x,y, listOf<Class<*>>(BoundingGrid::class.java), debugRaycast)
+
+                    ray.matches.firstOrNull()?.let { hit ->
+                        val node = hit.node as? SwingUiNode ?: return
+                        val hitPos = ray.initialPosition + ray.initialDirection.normalize(hit.distance)
+                        node.ctrlClick(hitPos)
+                    }
+                }
+            }
+        )
+        inputHandler?.addBehaviour(
+            "dragObject", object : DragBehaviour {
+                override fun init(x:Int, y: Int) {
+                    val ray = cam.getNodesForScreenSpacePosition(x,y, listOf<Class<*>>(BoundingGrid::class.java), debugRaycast)
+
+                    ray.matches.firstOrNull()?.let { hit ->
+                        val node = hit.node as? SwingUiNode ?: return
+                        val hitPos = ray.initialPosition + ray.initialDirection.normalize(hit.distance)
+                        node.pressed(hitPos)
+                    }
+                }
+                override fun drag(x: Int, y: Int) {
+                    val ray = cam.getNodesForScreenSpacePosition(x,y, listOf<Class<*>>(BoundingGrid::class.java), debugRaycast)
+
+                    ray.matches.firstOrNull()?.let { hit ->
+                        val node = hit.node as? SwingUiNode ?: return
+                        val hitPos = ray.initialPosition + ray.initialDirection.normalize(hit.distance)
+                        node.drag(hitPos)
+                    }
+                }
+                override fun end(x: Int, y: Int) {
+                    val ray = cam.getNodesForScreenSpacePosition(x,y, listOf<Class<*>>(BoundingGrid::class.java), debugRaycast)
+
+                    ray.matches.firstOrNull()?.let { hit ->
+                        val node = hit.node as? SwingUiNode ?: return
+                        val hitPos = ray.initialPosition + ray.initialDirection.normalize(hit.distance)
+                        node.released(hitPos)
+                    }
+                }
+            }
+        )
+        inputHandler?.addKeyBinding("ctrlClickObject", "2")
+        inputHandler?.addKeyBinding("dragObject", "1")
+    }
+
+
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
```

Comment:
Kdoc missing

Reply:
Same question as above? None of the examples have a doc atached to the main function call